code/planning/src/local_planner/utils.py

```
@@ -216,26 +217,28 @@
-    # Filter for potential recognition of ega vehicle front hood
-    filtered_cars_in_front = cars_in_front[np.where(cars_in_front[:, 1] > 0.7)]
-    if filtered_cars_in_front.size == 0:
-        # no car in front
-        return None
     # Return nearest car
-    return filtered_cars_in_front[np.argmin(filtered_cars_in_front[:, 1])]
+    min_object_in_front = cars_in_front[np.argmin(cars_in_front[:, 1])]
+    if oncoming:
+        # As the overtaking / cruising step does not work properly at the moment this
+        # is a workaround.
+        if min_object_in_front[1] > 9.0:
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)